docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -13,12 +13,13 @@ aliases:
 **Prerequisites**
 - *vmanomaly* is a part of enterprise package. You can get license key [here](https://victoriametrics.com/products/enterprise/trial) to try this tutorial.
 - In the tutorial, we'll be using the following VictoriaMetrics components:
-  -  [VictoriaMetrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) (v.1.93.2)
-  -  [vmalert](https://docs.victoriametrics.com/vmalert.html) (v.1.93.2)
-  -  [vmagent](https://docs.victoriametrics.com/vmagent.html) (v.1.93.2)
+  -  [VictoriaMetrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) (v.1.96.0)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html) (v.1.96.0)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html) (v.1.96.0)
```

Comment:
Are there any dependencies why you want to specify the version explicitly?

Reply:
No dependencies. Should I delete version?

Reply:
our intention here was to keep the guide reproducible as long as possible

as an alternative, we can rely on `:latest` tags in docker compose for vm, vmalert, vmagent if there is a little risk of breaking changes in their configs or way they operate, then we can drop these explicit versions from guide;

Reply:
> as an alternative, we can rely on :latest

No, this is a bad idea.